tests/components/conversation/test_default_agent.py

```diff
@@ -44,3 +44,15 @@ async def test_hidden_entities_skipped(
     assert len(calls) == 0
     assert result.response.response_type == intent.IntentResponseType.ERROR
     assert result.response.error_code == intent.IntentResponseErrorCode.NO_INTENT_MATCH
+
+
+async def test_cant_turn_on_sun(hass: HomeAssistant, init_components) -> None:
```

Comment:
We should stop adding tests to this file. This file should only be used for the forwarding of requests to agents as specified in this file (does the WS work, does it handle errors, do the services works).

This file should not be used to test the default agent (that's test_default_agent) and it should also not be used to test intent internals. In that case, use `helpers/test_intent.py`. This test would fall in the latter category, and can just test directly using the intent.